src/backend/lib/config.js

```diff
@@ -1,3 +1,12 @@
 const dotenv = require('dotenv');
+const { logger } = require('../utils/logger');
 
-dotenv.config();
+const result = dotenv.config();
+
+if (result.error && logger) {
+  logger.error(
+    '\n\n\t💡  It appears that you have not yet configured a .env file.',
+    '\n\t   Please refer to our documentation regarding environment configuration:',
+    '\n\t   https://github.com/Seneca-CDOT/telescope/blob/master/docs/CONTRIBUTING.md\n'
+  );
```

Comment:
In the future, I think it'd be a good idea to add something like [process.exit(1)](https://stackoverflow.com/a/5266239/8628446) here.